packages/st2/debian/st2.links

```diff
@@ -2,5 +2,6 @@ opt/stackstorm/st2/bin/st2 usr/bin/st2
 opt/stackstorm/st2/bin/st2-trigger-refire usr/bin/st2-trigger-refire
 opt/stackstorm/st2/bin/st2-rule-tester usr/bin/st2-rule-tester
 opt/stackstorm/st2/bin/st2-register-content usr/bin/st2-register-content
+opt/stackstorm/st2/bin/st2-apply-rbac-definitions usr/bin/st2-apply-rbac-definitions
```

Comment:
Where do we do that for RHEL?

I quickly glanced over the RPM packaging metadata and I couldn't find it.